modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java

```diff
@@ -238,13 +234,11 @@ public void testRollingFileAppender() throws Exception {
 					"test file name should be " + fileName, matcher.matches());
 
 				try (InputStream inputStream = url.openStream()) {
-					byte[] bytes = _getBytes(inputStream);
-
-					content = new String(bytes, StringPool.UTF8);
+					content = StreamUtil.toString(inputStream);
```

Comment:
As I said, you can ignore my change to use petra `StreamUtil`, and reconsider how you read the files.